objectivec/GPBDescriptor.h

```diff
@@ -122,6 +122,7 @@ typedef NS_ENUM(uint8_t, GPBFieldType) {
 - (nullable NSString *)enumNameForValue:(int32_t)number;
 - (BOOL)getValue:(nullable int32_t *)outValue forEnumName:(NSString *)name;
 
+- (BOOL)getValue:(int32_t *)outValue forEnumTextFormatName:(NSString *)textFormatName;
```

Comment:
Looks like the code support outValue being nullable.

Also list this after -textFormatNameForValue: (following the ordering above where getValue:... comes after the other)